pyomo/solvers/tests/models/batchdes.py

```diff
@@ -0,0 +1,84 @@
+#  MINLP written by GAMS Convert at 11/29/17 00:11:13
```

Comment:
Would like to see a little bit more source information - lit reference to where this model is described?